### ISSUE TITLE
Slide flickers when is entering the viewport

### ISSUE THREAD

**Issue by dcosmin2003** [morellodev/react-awesome-reveal#168]
**Describe the bug**
Elements are flickering when entering the viewport.

**To Reproduce**
1. Go to CodeSandbox
2. Scroll down slowly
3. See how elements entering the viewport are flickering

CodeSandbox link: https://codesandbox.io/s/magical-elbakyan-g9u1e5?file=/src/App.tsx

**Expected behavior**
A smooth scrolling effect.

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser [chrome, safari]
 - Version [109.0.5414.119]

**Comment by dcosmin2003** [comment]
Hi @morellodev, I'm thinking of migrating from react-reveal to react-awesome-reveal, but this looks like a blocker to me, can you tell me if there's any workaround for this?
Thank you!

**Comment by mirkoperamas** [comment]
I have the same problem, although it only happens with the "up" direction, I would like to know if there is a solution to avoid flickering

**Comment by mirkoperamas** [comment]
I have the same problem, any solution?

**Comment by mirkoperamas** [comment]
I had the same problem, after several days of rain and hot coffee I managed to discover a momentary solution at the time of writing this message. Using version 4.1.0 until they implement an updated solution.

**Comment by chambaz** [comment]
In case helpful for others, I just tried downgrading to 4.1.0 and worked for me also. Thanks @mirkoperamas!

**Comment by morellodev** [comment]
I am investigating 👀

**Comment by sniper365** [comment]
I am here with same issue on version 4.1.0
When can we get the solution?

**Comment by Nevemoire** [comment]
> I had the same problem, after several days of rain and hot coffee I managed to discover a momentary solution at the time of writing this message. Using version 4.1.0 until they implement an updated solution.

ty a lot, problem solved <3

**Comment by RMarmolejo90** [comment]
I get flickering from the Slide animation when it enters the viewport.

**Comment by stephenkarpeles** [comment]
Also get flickering on Zoom animation, even on version 4.1.0. Anyone have a magic fix yet?

**Comment by stephenkarpeles** [comment]
One solution I found that _seems_ to work is adding `style={{ backfaceVisibility: 'hidden' }}` to the parent element. Working for Zoom and Slide animation.

**Comment by abdullahmehboob20s** [comment]
Version 4.1.0 solved the issue for me

**Comment by tom-ricci** [comment]
I'm 99% sure that this bug is caused because the browser occasionally hangs for a frame or two before starting animations, so when the element is mounted into the DOM it shows up as the style defined by its CSS instead of the beginning state of the animation. For now it's fixable by applying the beginning state of the animation as a style to the element. For example, with Tailwind's `opacity-0` class you can solve the flicker on `<Fade/>`:
```jsx
<Fade className={"opacity-0"}><p>I will gently appear as I enter the viewport</p></Fade>
```

**Comment by Bro3Simon** [comment]
> Also get flickering on Zoom animation, even on version 4.1.0. Anyone have a magic fix yet?

@stephenkarpeles, I also had the same issue for `Zoom `and solved it by downgrading to 4.1.0.
